deny.toml

```diff
@@ -32,6 +32,7 @@ confidence-threshold = 0.8
 # See https://spdx.org/licenses/ for list of possible licenses
 # [possible values: any SPDX 3.7 short identifier (+ optional exception)].
 allow = [
+    "CC0-1.0",
```

Comment:
@onbjerg @mattsse I  have to allowed this license because `interprocess` use a library which is under it.

Reply:
can you add it under `exceptions` like the other `CC0-1.0` deps we have?

Reply:
Done.